store/src/java-test/com/zimbra/cs/imap/ImapLoadBalancingMechanismTest.java

```diff
@@ -84,6 +84,9 @@ public void ClientIpHashMechanismIpHashFromPool()
         Assert.assertEquals(s1, mech.getImapServerFromPool(req, pool));
         headers.put(ImapLoadBalancingMechanism.ClientIpHashMechanism.CLIENT_IP, "127.0.0.4");
         Assert.assertEquals(s2, mech.getImapServerFromPool(req, pool));
+        headers.put(ImapLoadBalancingMechanism.ClientIpHashMechanism.CLIENT_IP, "192.168.56.1");
+        req = new MockHttpServletRequest(null, null, null, 123, "127.0.0.1", headers);
+        Assert.assertEquals(s1, mech.getImapServerFromPool(req, pool));
```

Comment:
That is a good point.  Would we want to start doing this kind of update anytime a test file is updated?  Could be a lot of work to update all of the tests in the file in this fashion.